v1/linkml-schemas/credentials.yaml

```diff
@@ -73,31 +86,68 @@ slots:
     range: PersonWithOnlyCoreAttributesClass
   iss:
     range: uri
+    description: The `iss` (issuer) claim identifies the principal that issued the JWT. The processing of this claim is generally application specific. [[RFC 7519](https://www.rfc-editor.org/rfc/rfc7519), [Section 4.1.1](https://www.rfc-editor.org/rfc/rfc7519#section-4.1.1)]
```

Comment:
I'm not sure that the "application specific" thing is massively helpful for us to include here, but I guess there subtle shades between JWTs in OAuth vs JWTs that are VCs.

I do think we should capture something about validation of issuers against key material in *our* system, as that process is an important part of our security architecture (ie checking that `iss` is as expected given `kid`), especially in the context of verifiers that wrangle credentials from multiple issuers (ie SPOT and IPV Core). See <https://govukverify.atlassian.net/browse/SPT-131>.

Gonna play around with some wording here...

Reply:
Did you manage to get anywhere with proposing some wording here? - would be good to try and get this merged if we can.